ledger-bridge.js

```diff
@@ -11,6 +12,25 @@ const BRIDGE_URL = 'ws://localhost:8435';
 const TRANSPORT_CHECK_DELAY = 1000;
 const TRANSPORT_CHECK_LIMIT = 120;
 
+const serializeError = (error) => {
+  if (error instanceof TransportStatusError) {
+    return {
+      message: error.message,
+      name: error.name,
+      stack: error.stack,
+      text: error.text,
```

Comment:
Actually, I was checking the `TransportStatusError` type, and I could not not see any `.text` field, do we really need it here?